File: api/enums/system/error_code_enum.py
from enum import Enum


class ErrorCodeEnum(Enum):
    # chat
    CHAT_USER_CANCELLED_ERROR = 1000
    # general
    INTERNAL_SERVER_ERROR = 5000
    # agents
    MODEL_NOT_SUPPORTED_ERROR = 6000
    MESSAGE_ROLE_NOT_SUPPORTED_ERROR = 6001
    # templates
    TEMPLATE_NOT_FOUND_ERROR = 2000

File: web/enums/common/ErrorCodeEnum.ts
enum ErrorCodeEnum {
  // general
  UnknownError = 1000,

  // builds
  TemplateNotFoundError = 2000,

  // authentication
  UnauthorizedError = 4001,
  ForbiddenError = 4003,

  // network
  InternalServerError = 5000,
  ParsingError = 5001,
}

export default ErrorCodeEnum;
